tests/test_polaraveraging.py

```diff
@@ -1,15 +1,13 @@
 """Tests for polaraveraging.py"""
-import clmm
-import clmm.polaraveraging as pa
-from numpy import testing
 import numpy as np
+from numpy import testing
 from astropy.table import Table
-import os
-import pytest
 
+import clmm
+import clmm.polaraveraging as pa
 
-rtol = 1.e-7
-atol = 1.e-7
+RTOL = 1.e-7
+ATOL = 1.e-7
```

Comment:
Lets convert this to `TOLERANCE={'atol':1.e-7, 'rtol':1.e-7}` and then we can just `**TOLERANCE` into the asserts.

Reply:
done